### ISSUE TITLE
CodeAlly CodeRoad

### ISSUE THREAD

**Issue by texas2010** [freeCodeCamp/freeCodeCamp#47304]
### Describe the Issue

I feel the font size is too small for me and I can't change it in the CodeAlly CodeRoad.  this part needs to allow us to change font size when we need to do.

### Affected Page

https://www.freecodecamp.org/learn/relational-database/learn-bash-by-building-a-boilerplate/build-a-boilerplate

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

N/A

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by moT01** [comment]
What specifically is too small @texas2010?

It looks maybe a little small to me:
![Screen Shot 2022-08-17 at 11 21 10 AM](https://user-images.githubusercontent.com/20648924/185191814-50f4bd38-5b69-4841-97bb-9b1e110f1716.png)

Although, not much smaller than the default size in that getting started tab. You can zoom the browser in for now if you want it bigger.

**Comment by hbar1st** [comment]
I agree with this. I am constantly having to zoom in and out again to see what is going on (and sometimes blindly copy the characters because I cannot tell and I from an L)

**Comment by hbar1st** [comment]
I am middle-aged and need bifocals. The text of the instructions is also too small for me. I end up zooming in with the browser settings especially on instructions that have tiny one or two character commands. (Is it an l or an I ?) 
Needs to be one font size larger in my estimation.
